tree/ntuple/v7/src/RMiniFile.cxx

```diff
@@ -1152,6 +1152,10 @@ ROOT::Experimental::Internal::RNTupleFileWriter::Recreate(std::string_view ntupl
       flags |= O_DIRECT;
    }
    int fd = open(std::string(path).c_str(), flags, 0666);
+   if (fd == -1) {
+      throw RException(R__FAIL("open(" + std::string(path) + ", " + std::to_string(flags) +
+                               ", 0666) failed with errno: " + strerror(errno)));
```

Comment:
I would be even less technical and say
```suggestion
      throw RException(R__FAIL(std::string("open failed: ") + strerror(errno)));
```
(in line with other exceptions in this file)